src/app/client/src/app/modules/profile/components/profile-badge/profile-badge.component.ts

```diff
@@ -13,6 +13,10 @@ export class ProfileBadgeComponent implements OnInit {
  * Reference of User Profile interface
  */
   userProfile: IUserProfile;
+  viewMore = true;
+  defaultLimit = 3;
+  limit = 3; // config
+  badgeData: any;
```

Comment:
Not as per latest change in angular-migration branch.

Expected:
Unused variable badgeData is removed.